typescript/appconfig-hosted-configuration-lambda-extension/README.md

```diff
@@ -0,0 +1,58 @@
+# AWS AppConfig Hosted Configuration with Lambda Extension
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
```

Comment:
This actually isn't considered a "stable" module since only the L1 containing Cfn resources exists. You can see the banner [here](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-appconfig-readme.html) and explanation of reasoning [here](https://docs.aws.amazon.com/cdk/latest/guide/reference.html)